### ISSUE TITLE
Local/Remote IOM connection: Switching SAS Profiles now affects all VSCode instances

### ISSUE THREAD

**Issue by id2-nicoDt** [sassoftware/vscode-sas-extension#986]
**Describe the bug**
Since v1.9.0, switching the SAS profile in one VSCode window switches the profile of all other VSCode instances and terminates the connections (in our case IOM remote connections). 

**Steps to reproduce**
1. Define several SAS Profiles (2 IOM remote, but might affect other types)
2. Open 2 VS Code instances 
3. Run SAS Code in one instance, connecting to one SAS Profile
4. Go to the second VS Code instance, switch the SAS Profile
5. Observe that the profile has also been changed in the first instance, terminating (I hope) the previous connection

**Expected behavior**
SAS Profiles should be independent between VS Code instances, and it should be possible to have several opened at the same time.
This is the behavior that I experienced in the past.

**Screenshots**
None

**Environment (please complete the following information):**
Client OS: Windows 10 Pro
Extension version: v1.9.0
SAS version: SAS 9.4 (remote - IOM)

**Additional context**
I work with several sas servers on different continents and therefore I have several SAS Profiles defined.
When working simultaneously on projects set on different servers, I used to open several VS Code windows and connected each to the appropriate server/SAS Profile (using IOM remote). This issue does not permit that anymore, and is especially problematic as it increases the risk of running code on the wrong server.

**Comment by scnwwu** [comment]
I don't see any change related to this. What version does it work for you? You can try previous versions by clicking the "Install Another Version..." from the Extensions pane.
The profiles settings are stored in VS Code settings.json file. By default it is in your home directory so applies to all instances. You can manually create [workspace settings](https://code.visualstudio.com/docs/getstarted/settings#_workspace-settings). Or you can create [VS Code profiles](https://code.visualstudio.com/docs/editor/profiles).

**Comment by id2-nicoDt** [comment]
Thanks for the suggestions!
Checking previous versions it appears that either I misremembered (likely) or for some reason VSCode changed its way of handling workspaces.
I had already defined workspaces, but following your input I went further to set the workspace settings for SAS connection profiles. This does work.

I will close this as it isn't an issue, and create a feature request for either a more front-facing option for workspace SAS profiles, or an update to the 'Profiles' section of the readme so that it spells out that by default the profiles will be shared among workspaces.
